Yosemite/YosemiteTests/Mockups/Networking/Remote/MockProductReviewsRemote.swift

```diff
@@ -0,0 +1,41 @@
+
+import Foundation
+import Networking
+
+import XCTest
+
+/// Mock for `ProductReviewsRemote`.
+///
+final class MockProductReviewsRemote: ProductReviewsEndpointsProviding {
+    private struct ResultKey: Hashable {
+        let siteID: Int64
+        let reviewID: Int64
+    }
+
+    /// The results to return based on the given arguments in `loadProductReview`
+    private var productReviewLoadingResults = [ResultKey: Result<ProductReview, Error>]()
+
+    /// Set the value passed to the `completion` block if `loadProductReview()` is called.
+    ///
+    func whenLoadingProductReview(siteID: Int64, reviewID: Int64, thenReturn result: Result<ProductReview, Error>) {
```

Comment:
nit: would it be more clear if `whenLoadingProductReview` is in an extension so that the protocol implementation methods are separate from the methods that mock the network response?